scripts/Dockerfile.ble-sim

```diff
@@ -27,6 +27,10 @@ WORKDIR /ble-sim/
 RUN scripts/download-deps.sh
 RUN scripts/build.sh
 
+WORKDIR /
+#RUN git clone --branch v1.2-branch-sdk-update https://github.com/IOTechSystems/xrt-examples
+RUN git clone --branch XRT-1193-branch https://github.com/IOTechSystems/xrt-examples
```

Comment:
Should the branch be changed before merge?

Reply:
Yeah sorry, didn't mean to leave that in: if everything else is good then I will fix that.